packages/techdocs-cli/src/commands/helpers.test.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command } from "commander";
+import { getValidPublisherConfig, Publisher } from './helpers';
+
+describe('getValidPublisherConfig', () => {
+  it ("should not allow unknown publisher types", () => {
+    const invalidConfig = {
+      publisherType: "unknown publisher"       
+    } as unknown as Command;
+
+    expect(getValidPublisherConfig.bind(null, invalidConfig)).toThrow();
```

Comment:
It's helpful for our user to have a clear error message, so I recommend including the message text in the error test:
```suggestion
    expect(getValidPublisherConfig.bind(null, invalidConfig)).toThrowError(
      `Unknown publisher type ${invalidConfig.publisherType}`
    );
```